lib/src/client/query_parameter.dart

```diff
@@ -29,17 +29,24 @@ class QueryParameter implements Comparable<QueryParameter> {
   String get value => values.first;
 
   /// Creates an instance by wrapping the single value in a [List].
-  QueryParameter(this.key, String value)
-      : assert(key != null && key.trim().isNotEmpty),
-        values = [value];
+  QueryParameter(this.key, String value) : values = [value] {
+    ArgumentError.checkNotNull(key);
```

Comment:
I guess this is for mixed mode?

Reply:
Yes, from go/dart-null-safety-faq#how-should-i-migrate-an-assertx-null-that-now-shows-as-unnecessary I like this option the most for mixed mode, though I don't have strong opinion